src/domains/LottoShop/index.js

```diff
@@ -0,0 +1,17 @@
+export class LottoShop {
+  static BASE_LOTTO_PRICE = 1_000;
+
+  #lottoMachine;
+  #lottoPrice;
+
+  constructor({ lottoMachine, lottoPrice = LottoShop.BASE_LOTTO_PRICE } = {}) {
+    this.#lottoMachine = lottoMachine;
+    this.#lottoPrice = lottoPrice;
+  }
+
+  buyLottos(price) {
+    const lottoCount = Math.floor(price / this.#lottoPrice);
+
+    return this.#lottoMachine.issueLottos(lottoCount);
+  }
+}
```

Comment:
```suggestion
export class LottoShop {
  static LOTTO_PRICE = 1_000;

  static buyLottos(price) {
    const lottoCount = Math.floor(price / this.LOTTO_PRICE);

    return LottoMachine.issueLottos(lottoCount);
  }
}
```

이렇게 static method를 통해 표현할 수 있답니다!

그런데 생각해보니 현재 모습을 유지할 때 "지점마다 로또의 가격이 달라질 수 있음" 이라는 요구사항이 있을 때 수용하기가 쉽겠군요 ㅎㅎ